packages/experimental/Tooltip/SPEC.md

```diff
@@ -2,4 +2,131 @@
 
 ## Background
 
-Please write here you documentation...
+A `Tooltip` is a component that displays additional information about another component. The information is displayed above or near the target. At this time, the tooltip is displayed at or near the mouse cursor (target), which is default Win32 behavior.
+
+## Sample Code
+
+Basic Example:
+
+```jsx
+const tooltip = (
+  <Tooltip content={defaultContent}>
+    <ButtonV1 accessibilityDescription={defaultContent}>Default Tooltip</ButtonV1>
+  </Tooltip>
+);
+```
+
+More examples on the [Test pages for Tooltip](../../../apps/fluent-tester/src/TestComponents/Tooltip). Instructions on running the tester app can be found [here](../../../apps/fluent-tester/README.md).
+
+## Visual Examples
+
+### Default Tooltip
+
+Win32:
+
+![Default tooltip](assets/image.png)
+
+```tsx
+const defaultContent = 'Tooltip shows relative to mouse cursor.';
+
+const tooltip = (
+  <Tooltip content={defaultContent}>
+    <ButtonV1 accessibilityDescription={defaultContent}>Default Tooltip</ButtonV1>
+  </Tooltip>
+);
+```
+
+## Variants
+
+### Positioning
+
+Not yet implemented.
+
+The `Tooltip` component will appear in the specified position relative to the anchor (target). In Win32, the positions are denoted by the DirectionalHint: `topLeftEdge`, `topCenter`, `topRightEdge`, `rightTopEdge`, `rightCenter`, `rightBottomEdge`, `bottomLeftEdge`, `bottomCenter`, `bottomRightEdge`, `leftTopEdge`, `leftCenter`, and `leftBottomEdge`.
```

Comment:
Can we align with FluentUI React and use the more react-style names? The names (which felt a bit win32 specific) was something I hard to hardcode into macOS Callout and didn't like
https://github.com/microsoft/fluentui/blob/4ad395276e01a2a04df8f0263e1af2467310bcfd/packages/react-components/react-positioning/src/types.ts#L244

```
export type PositioningShorthandValue =
  | 'above'
  | 'above-start'
  | 'above-end'
  | 'below'
  | 'below-start'
  | 'below-end'
  | 'before'
  | 'before-top'
  | 'before-bottom'
  | 'after'
  | 'after-top'
  | 'after-bottom';
  ```

Reply:
We had decided to align with the naming convention used in Callout to have consistency across the existing FURN controls. I'll make a note to get PM input on this though when we revisit Tooltip in the future.